src/Controls/src/Core/Platform/Windows/VisualElementTracker.cs

```diff
@@ -200,7 +208,17 @@ void HandleDragStarting(UIElement sender, Microsoft.UI.Xaml.DragStartingEventArg
 				}
 
 				var renderer = sender as IViewHandler;
-				var args = rec.SendDragStarting(renderer?.VirtualView);
+
+				Point position = Point.Zero;
+
+				if (e.OriginalSource is UIElement element)
+				{
+					var result = e.GetPosition(element);
+					position = new Point(result.X, result.Y);
+				}
+
+				var args = rec.SendDragStarting(renderer?.VirtualView, position);
```

Comment:
Are we ok with this taking a null as the first param?

Reply:
`SendDragStarting` does not really handle null elements. I added an extra statement there to throw if somehow we get a null drag source. @jsuarezruiz  thoughts?